packages/eas-json/src/utils.ts

```diff
@@ -1,9 +1,11 @@
 import { Platform } from '@expo/eas-build-job';
+import { Cache } from '@expo/eas-build-job/dist/common';
```

Comment:
Do not import like this, if it's not exported from the entry point then make the change in a package to export it.